packages/remix-azure-functions/package.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "@remix-run/azure-functions",
+  "description": "Azure Functions server request handler for Remix",
+  "version": "1.3.3",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/remix-run/remix",
+    "directory": "packages/remix-azure-functions"
+  },
+  "bugs": {
+    "url": "https://github.com/remix-run/remix/issues"
+  },
+  "dependencies": {
+    "@remix-run/node": "1.3.3"
+  },
+  "peerDependencies": {
+    "@azure/functions": "^1.2.3"
```

v3 is already available

```suggestion
    "@azure/functions": "^3.0.0"
```